src/components/FloatingSubscribeNewsletter.js

```diff
@@ -30,8 +30,9 @@ import mailHoverImg from './images/mail-share-hover.svg';
 import hasuraGray from './images/hasura-gray.svg';
 
 import hasuraBlue from './images/hasura-blue.svg';
+import { getUTMPagePathName } from '../utils/getUTMPagePathName';
```

Comment:
This file doesn't exist in this PR?

Reply:
The utils file `getUTMPagePathName` was added in #118